components/scream/tests/uncoupled/homme/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ CreateDynamicsLib("theta-l_kokkos"  4   72   35)
 
 set (NEED_LIBS ${dynLibName} scream_control scream_share)
 
-# Test Homme standalone
+# Test Homme stand alone
```

Comment:
Maybe okay to leave as "standalone". Standalone (or stand-alone) is an adjective, like "standalone HOMME", whereas to "stand alone" is a verb phrase.

Not a big deal, though.

Reply:
I was just trying to make it so that we use the same version. There was some "standalone" and some "stand_alone". I tossed a coin, and picked one.

Reply:
Done